datm/cime_config/stream_definition_datm.xml

```diff
@@ -236,7 +236,7 @@
       <file first_year="$DATM_YR_START" last_year="$DATM_YR_END">$DIN_LOC_ROOT/atm/cdeps/v1/$NEONSITE/%ym.nc</file>
     </stream_datafiles>
     <stream_datavars>
-      <var>PRECTmms Faxa_precn</var>
+      <!-- <var>PRECTmms Faxa_precn</var> -->   <!-- COMMENT THIS OUT WHEN RUNNING WITH PRISM-->
```

Comment:
It's important to construct this in a way that the user wouldn't have to comment out something like this by hand.

Reply:
Yes, I fully agree! Thank you for this comment. I think there will be a few components that I need to figure out how to switch on and off based on whether NEON or PRISM precipitation is being used. Eg, there are also some shell commands that can be deleted when PRISM is being used but are necessary with NEON (see comment in description of https://github.com/ESCOMP/CTSM/pull/1954 regarding NIWO DATM_YEAR_END shell commands)

Reply:
agreed @ekluzek can you or @jedwards4b  suggest name list changes or other ways to accomplishing this that are consistent with CTSM/CDEPS/CESM standards?  

For example, I can imagine passing a `--use_PRISM` flag in run_neon that sets up cases as Teagan has here, but I'm not sure if this is best accomplished with another set of duplicated usermods_dir (similar to NEON_FATES), or something else?

More I'm not really clear how to best pass this information to CDEPS so users don't have do do any of these external modifications?

Reply:
@TeaganKing and @wwieder can you post a current case you have setup that runs with PRISM? Either a case that you customized to do it, or a test case that's starting to use this new work.

I'd like to look that over a bit to think of some options, and then we should meet to talk about ways of doing this.

Reply:
You should add code in cdeps/datm/cime_config/buildnml to remove this line when prism is used.

Reply:
My first idea is that we add a new XML variable for DATM that controls this. But, yes this could be something in the CTSM user-mod directories, or an option that is triggered for the datm buildnml. It would be good to brainstorm options, and then go through and decide which approach is the best.

Reply:
Do we not already have something that indicates that PRISM is used?   If so I don't think that we need another variable.

Reply:
It looks like right now there isn't a single XML variable that says "run with PRISM.PRECIP". What is currently being done is that PRISM requires an extra stream for PRECIP and you remove PRECIP from the main NEON stream. So we need a way to trigger which way you want this to go in some fashion. That's why I think this needs some discussion...

Reply:
Hi @ekluzek , some examples of the current implementation can be found here: /glade/work/tking/ctsm_tking/tools/site_and_regional . That is correct that we are currently removing one data stream and adding another manually; we'd like to use some sort of keyword that triggers one method or the other.